shared/uavobjectdefinition/stabilizationsettings.xml

```diff
@@ -4,16 +4,16 @@
 	<field name="RollMax" units="degrees" type="uint8" elements="1" defaultvalue="55" limits="%BE:0:180"/>
 	<field name="PitchMax" units="degrees" type="uint8" elements="1" defaultvalue="55" limits="%BE:0:180"/>
 	<field name="YawMax" units="degrees" type="uint8" elements="1" defaultvalue="35" limits="%BE:0:180"/>
-	<field name="ManualRate" units="degrees/sec" type="float" elementnames="Roll,Pitch,Yaw" defaultvalue="150,150,150" limits="%BE:0:500,%BE:0:500,%BE:0:500"/>
+	<field name="ManualRate" units="degrees/sec" type="float" elementnames="Roll,Pitch,Yaw" defaultvalue="150,150,150" limits="%BE:0:1000,%BE:0:1000,%BE:0:1000"/>

```

Comment:
I think the big issue is that the red looks like a big scary error but isn't well documented.  Maybe instead of changing the point at which it triggers we should just add a more obvious explanation of WHY it's being flagged.  Seems like it's good functionality, just poorly explained.